object/basic.go

```diff
@@ -214,3 +226,21 @@ func UpdateLatestSyncedRecordId(id int) bool {
 
 	return affected != 0
 }
+
+// GetOnlineMemberNum returns online member num.
+func GetOnlineMemberNum() int {
+	if onlineMemberNum == 0 {
```

Comment:
Maybe `if` cause the bug.

Reply:
@oceanlvr this PR is done. Plz raise a new PR.

Reply:
@hsluoyz Ok, I open a new PR #113